src/crypto/ecdh.rs

```diff
@@ -140,7 +140,7 @@ pub fn decrypt(priv_key: &ECDHSecretKey, mpis: &[Mpi], fingerprint: &[u8]) -> Re
         None => 0,
     };
 
-    let mut encrypted_session_key_vec: Vec<u8> = Vec::new();
+    let mut encrypted_session_key_vec: Vec<u8> = vec![0; encrypted_key_len];
     encrypted_session_key_vec.resize(encrypted_key_len, 0);
```

Comment:
no need for this line, if you initialize them above already

Reply:
`clippy::perf` suggested it but could change it back.

Reply:
Reverting the change causes Github actions to fail ie. [rpgp/actions/runs/5912769030](https://github.com/LiveDuo/rpgp/actions/runs/5912769030/job/16036499778).